ui/components/Page.tsx

```diff
@@ -32,7 +32,6 @@ export const Content = styled(Flex)`
   padding-bottom: ${(props) => props.theme.spacing.medium};
   padding-left: ${(props) => props.theme.spacing.large};
   padding-top: ${(props) => props.theme.spacing.medium};
-  padding-right: ${(props) => props.theme.spacing.large};
```

Comment:
I just added this back in here so the page wouldn't extend all the way off the screen: https://github.com/weaveworks/weave-gitops/commit/3d63640ab4b132a03696a31388bff030f1f2c061#diff-5172b3d6127736059f59dbba793deb31dfbed2b93c0211f9c5cb4c397607ed4aR35

Was that incorrect?

Reply:
When I deleted it the filter bar moved off the screen again - so I guess so? That padding pushes the filter bar to the left.